apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -84,6 +88,7 @@ private ApolloClient(Builder builder) {
     this.defaultHttpCacheControl = builder.defaultHttpCacheControl;
     this.defaultCacheControl = builder.defaultCacheControl;
     this.logger = builder.apolloLogger;
+    this.applicationInterceptors = builder.applicationInterceptors;
```

Comment:
Well, the initial thought was to wrap it in an unmodifiable list, but then I realized that we are not exposing the list of interceptors in any way to the clients, so it might as well just remain a simple List.